_posts/2020-02-17-week-3.md

```diff
@@ -30,6 +30,14 @@ View our detailed lesson plan [here](https://drive.google.com/open?id=1vYh85Un-C
 - **Worksheet** (30 min): Log into Scratch accounts and work on the worksheet
 - **Wrap up** (5 min): See how loops fit with the maze activity
 
+### Video Materials
+We've produced exciting videos that go over the topics outlined in our lesson plan. These can serve as inspiration for how you could conduct your lessons, be used alongside your own explanations to help students visualize certain concepts, or even act as their own stand-alone lesson.

```

Comment:
change intro: We've produced exciting videos that go over the topics outlined in our lesson plan. These can serve as inspiration for how you could conduct your lessons. If you or the students don’t have time to watch the videos...check out our quick cheat sheet: 
https://drive.google.com/file/d/1PIynUwSKo55HSybjRD5XStQBIu6G-o5h/view?usp=sharing